src/components/Tabs/Settings/BackdropSettings.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { type ReactElement, useMemo } from "react";
```

Comment:
Compare this (and all other files) against `SettingsTab.tsx` + turn off whitespace for an easier review!